jenkins-pipelines/Jenkinsfile

```diff
@@ -0,0 +1,54 @@
+/*
+* CI Pipeline to build, publish and run the ML Training Pipeline.
+* Pipeline uses docker image as an agent.
+*/
+pipeline {
+    agent {
+        docker {
+            //Azure CLI commands require root privileges
+            image 'mcr.microsoft.com/mlops/python:latest'
+            args "--user root --privileged"
+        }
+    }
+    stages {
+        stage('Build, Publish and Run AML Pipeline') {
+            environment {
+                WORKSPACE_NAME = 'swsundar-ml'
```

Comment:
@sundar-swetha  - if it isn't a lot of work (and I believe it isn't) I think it is better to set these as global variables in the job (and include these values default values) and reference them from the pipeline, keep it clean.

Reply:
Done.